### PR TITLE
Normalize all `scope` and `scopeSeparator` properties

### PR DESCRIPTION
**Pull request by lucasmichot**
Normalize all `scope` and `scopeSeparator` properties

### CODE REVIEW

File: src/Line/Provider.php [hunk 1]
```diff
@@ -12,16 +12,12 @@ class Provider extends AbstractProvider
...
-     * @var string
+     * {@inheritdoc}
      */
     protected $scopeSeparator = ' ';
 
     /**
-     * The scopes being requested.
-     *
-     * @var array
+     * {@inheritdoc}
```
Comment (atymic) [inline src/Line/Provider.php:20]: Why is this needed? This is just for code doc generation right, which we dont do?

Comment (lucasmichot) [reply src/Line/Provider.php:20]: > This is just for code doc generation right, which we dont do?
It is

Another approach would be be to remove all the docblocks

Comment (lucasmichot) [reply src/Line/Provider.php:20]: @atymic I can remove them all, just let me know

Comment (lucasmichot) [reply src/Line/Provider.php]: I've removed them; see 3de8aca127600ef506df292e3ca66d06a62cb202

**Review by atymic — APPROVED** [review]

**Review by atymic — APPROVED** [review]
